cli/flox/src/commands/search.rs

```diff
@@ -41,15 +41,15 @@ pub struct ChannelArgs {}
 // Search for packages to install
 #[derive(Bpaf, Clone)]
 pub struct Search {
-    /// print search as JSON
+    /// Display search results as a JSON array
     #[bpaf(long)]
     pub json: bool,
 
     /// Print all search results
     #[bpaf(short, long)]
     pub all: bool,
 
-    /// The package to search for in the format '<pkg>[@<semver-range>]' using 'node-semver' syntax.
+    /// The package to search for in the format '<pkg-path>[@<semver-range>]' using 'node-semver' syntax.
```

Comment:
nonblocking: I think we should clarify that `pkg-path` can actually be part of the `pkg-path`. (I think we want to require a complete match against `pkg-path` when there isn't an `@`, but that requires a code change https://github.com/flox/flox/issues/816#issuecomment-1904955290)

Reply:
I'd prefer to leave that specific part of the current behavior undocumented since we expect to change it.